keps/sig-api-machinery/3157-watch-list/README.md

```diff
@@ -660,8 +661,19 @@ We expect no non-infra related flakes in the last month as a GA graduation crite
 - The Feature is implemented behind `WatchList` feature flag
 - Initial e2e tests completed and enabled
 - Scalability/Performance tests confirm gains of this feature
+- Add support for watchlist to APF
+
+#### Beta
 - Metrics are added to the kube-apiserver (see the [monitoring-requirements](#monitoring-requirements) section for more details)
 - Implement `SendInitialEvents` for `watch` requests in the etcd storage implementation
+- The feature is enabled for kube-apiserver and kube-controller-manager 
+- The generic feature gate mechanism is implemented in client-go. 
```

Comment:
I don't think we need to describe all feature of the new mechanism that will be implemented in the `client-go`
I described how it will be used by `kcm` in `Can the feature be disabled once it has been enabled ` section.